ecs-cli/modules/clients/aws/ecs/client.go

```diff
@@ -35,7 +35,7 @@ type ProcessTasksAction func(tasks []*ecs.Task) error
 
 // ECSClient is an interface that specifies only the methods used from the sdk interface. Intended to make mocking and testing easier.
 type ECSClient interface {
-	// TODO: Modify the interface and tbe client to not have the Initialize method.
```

Comment:
lol. Thanks for continuing to fix all of the weird spelling errors in the CLI! :)